### ISSUE TITLE
Install PoolParty on the server

### ISSUE THREAD

**Issue by dw8547** [devinit/devdatawarehouse#20]
Prepare (configure) the server in preparation for the installation.
Check the software download link.
Set up Oracle (Sun) Java Runtime Environment 7.
Set up Tomcat7.

**Comment by dw8547** [comment]
Mark Kahn has now given some paperwork to Leighton James. Leighton James is trying to get in touch with you Bill, he seems unsure about what to do/nervous about the amount of $. Checking, so there is no miscommunication about this, will I or Leighton be installing PoolParty on the server?

**Comment by dw8547** [comment]
Installed PoolParty on juds.devinit.org.
Cannot start PoolParty.
Contacted SWC (FH). FH forwarded query to MS.

Debugged to an official bug (possibly), issue between Ubuntu 14.04 and tomcat7:
https://bugs.launchpad.net/ubuntu/+source/tomcat7/+bug/1232258

**Comment by dw8547** [comment]
The PPT Installation Guide was not clear on whether the user needed to install and configure Tomcat herself/himself. This is what I assumed from these instructions:
***
This installation instructions assume that a PoolParty standard installation will be made using the following options:
-Tomcat is running as localhost using port 80
***
I installed Tomcat whereas the PPT installer is shipped with a preconfigured Tomcat. This is why I could not access PPT via the browser yesterday (12/05/2015) after I had installed and started PPT, i.e., I did it manually when the PPT installer needs to do it itself. 

FH has updated the manual to:
***
-PoolParty Tomcat will be running as localhost using port 80
***

Today, I wiped the server hosting PPT, reconfigured it and installed PPT from scratch.

It works! And can be accessed via a web browser as: http://185.52.0.7/PoolParty/.

Woohoo!
